code/game/objects/structures/janicart.dm

```diff
@@ -1,219 +1,3 @@
-//old style PIMP-CART
-/obj/structure/stool/bed/chair/janicart
-	name = "janicart"
-	desc = "A brave janitor cyborg gave its life to produce such an amazing combination of speed and utility."
-	icon = 'icons/obj/vehicles.dmi'
-	icon_state = "pussywagon"
-	anchored = 0
-	density = 1
-	var/obj/item/weapon/storage/bag/trash/mybag = null
-	var/callme = "pimpin' ride"	//how do people refer to it?
-	var/move_delay = 0
-	var/floorbuffer = 0
-	var/keytype = /obj/item/key/janitor
-
-/obj/structure/stool/bed/chair/janicart/New()
-	handle_rotation()
-
-/obj/structure/stool/bed/chair/janicart/Move(a, b, flag)
-	..()
-	if(floorbuffer)
-		var/turf/tile = loc
-		if(isturf(tile))
-			tile.clean_blood()
-			if (istype(tile, /turf/simulated/floor))

```

Comment:
this section right here is the dirt thing that you need to still include for the janicart